### ISSUE TITLE
How to use it with 

### ISSUE THREAD

**Issue by tshaiman** [blacktop/seccomp-gen#1]

**Comment by tshaiman** [comment]
Hi , 
I have a container that I want to auto-generate the seccomp profile as your tutorial shows.
however the container does not run in standalone mode and is getting its information from other services.
so one way we can ran it outside k8s which is its native runtime execution is a mock using docker-compose.

Question is , how can I redirect the output and use 'scgen' when I'm from "docker-compose up" and not "docker run" ? 

in other words , what is the equivalent to this command :
docker run --rm --security-opt seccomp=unconfined img1:scgen 2>&1 | scgen -verbose

when img1:scgen is just 1 out of 3 containers in Docker-compose  ?

thank you
amazing Project.
